File: solutions/72-rafidhaque/src/Circle.java
import javafx.scene.paint.Color;
import javafx.scene.shape.Arc;
import javafx.scene.shape.ArcType;
import javafx.scene.shape.Shape;

public class Circle {
    Arc arc;
    double x, y, diameter;

    public Circle(double centerX, double centerY, double diameter) {
        this.x = centerX;
        this.y = centerY;
        this.diameter = diameter;
        arc = new Arc(x, y, diameter, diameter, 270, 360);
        arc.setFill(Color.TRANSPARENT);
        arc.setStroke(Color.BLACK);
        arc.setType(ArcType.ROUND);
    }

    public Shape draw() {
        return arc;
    }

}

File: solutions/09-rafid/age of villagers/age of villagers/ageofvillagers/Circle.cs
using System;

public class Circle: Shape
{
    private int x, y, radius;
    public Circle(int x, int y, int radius)
    {
        this.x = x;
        this.y = y;
        this.radius = radius;

    }
    public string Draw()
    {
        return "Drawing a circle";
    }
}
